apps/docs/azure-functions/src/lib/auth.ts

```
@@ -2,89 +2,246 @@
+      },
+    };
+  }
+
+  const userId = claims.sub || claims.oid;
   if (!userId) {
+    return {
+      authenticated: false,
+      userId: null,
+      error: {
+        status: 401,
+        body: { error: "Token missing user identifier", code: "unauthenticated" },
```

ESLint

[error] Replace `·error:·"Token·missing·user·identifier",·code:·"unauthenticated"` with `⏎··········error:·"Token·missing·user·identifier",⏎··········code:·"unauthenticated",⏎·······`

(prettier/prettier)